en/micro-integrator/docs/use-cases/examples/template_examples/using-endpoint-templates.md

```diff
@@ -28,50 +28,117 @@ For example, let's say we have two default endpoints with following hypothetical
+<template xmlns="http://ws.apache.org/ns/synapse" name="ep_template">
+   <parameter name="name"/>
+   <parameter name="uri"/>
+   <parameter name="codes"/>
+   <parameter name="factor"/>
+   <parameter name="retries"/>
+   <endpoint name="$name">
+      <address uri="$uri">
+         <suspendOnFailure>
+            <errorCodes>$codes</errorCodes>
+            <progressionFactor>$factor</progressionFactor>
+         </suspendOnFailure>
+         <markForSuspension>
+            <retriesBeforeSuspension>$retries</retriesBeforeSuspension>
+            <retryDelay>1</retryDelay>
+         </markForSuspension>
+      </address>
+   </endpoint>
 </template>
 ```
 
 !!! Note
-    - The Endpoint template uses parameters as inputs. These parameters are then refered using the `$` prefix within the template. Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). For example, a parameter named `codes` can be referred by `$codes`.
+    - The endpoint template uses parameters as inputs. These parameters are then refered using the `$` prefix within the template. Unlike sequence templates, endpoint templates are always parameterized using `$` prefixed values (not XPath expressions). For example, a parameter named `codes` can be referred by `$codes`.
     - `$name` and `$uri` are default parameters that a template can use anywhere within the endpoint template (usually as parameters for endpoint name and address attributes).
 
 Since we have a template defined, we can use template endpoints to create two concrete endpoint instances with different parameter values for this scenario. This is shown below.
 
-``` java tab='Endpoint 1'
-<endpoint name="ep1" template="ep_template" xmlns="http://ws.apache.org/ns/synapse">
-  <parameter name="codes" value="10001,10002" />
-  <parameter name="factor" value="1.0" />
-  <parameter name="retries" value="5" />
+``` xml tab='Endpoint 1'
+<endpoint name="ep1" template="ep_template" uri="http://run.mocky.io/v3/8ca6aa42-ee0a-47a8-8007-e93abbb95b87">
+  <parameter name="codes" value="10001,10002"/>
+  <parameter name="retries" value="2"/>
+  <parameter name="factor" value="1.0"/>
 </endpoint>
 ```
 
-``` java tab='Endpoint 2'
-<endpoint name="ep2" template="ep_template" xmlns="http://ws.apache.org/ns/synapse">
-  <parameter name="codes" value="10001,10003" />
-  <parameter name="factor" value="2.0" />
-  <parameter name="retries" value="3" />
+``` xml tab='Endpoint 2'
+<endpoint name="ep2" template="ep_template" uri="http://run.mocky.io/v3/c72a5cfd-871b-43fc-8202-54fa18097341">
+  <parameter name="codes" value="10001,10003"/>
+  <parameter name="retries" value="3"/>
+  <parameter name="factor" value="2.0"/>
 </endpoint>
 ```
+### Synapse configuration
+
+In this example, the endpoint template is configured to invoke the endpoints based on the API invocation. According to this configuration, we parameterise the endpoint name, URI, codes, retries and the factor.
```

Comment:
Use "you" instead of "we", and "parameterize" instead of "parameterise".

Reply:
Done